### ISSUE TITLE
FTA shows different unknown relations when changing root person of married person

### ISSUE THREAD

**Issue by sbankscharles** [ShammyLevva/FTAnalyzer#253]
**Version Number problem appears in**
8.5.1.0
In all fairness to you, this may or may not be a bug in FTAnalyzer but could be an indication of an issue with Family Tree Maker 2019.

I have attached my gedcom file from an export of Family Tree Maker where I am the home (root) person.  After loading the file in FTA you will see that there are no unknown relationships.  Now, if you change the root person to my husband Jonathan Miles Ray you will have 297 unknown relationships.  The difference appears to be that when I'm the home person all of Jon's ancestors are related to me by marriage.  When Jon is the home person none of my ancestors are related by marriage to him.  

In Family Tree Maker we are set as 'ongoing spouse'.  His mother is my mother-in-law and my mother is his mother-in-law.

Typically, I am always the home person in FTM but I had issues with relationships on his side of the family recently and I thought I'd switch to him being the root.  My point is, I don't know if this has always been like this or not.  Back in November/December with me as the home person I worked to get unknown relationships to 0.  Something happened in FTM and that number jumped to 17

![charles home](https://user-images.githubusercontent.com/8715213/149543494-05aaf326-06fa-4a89-9790-7894e045fa86.JPG)
![Jon home](https://user-images.githubusercontent.com/8715213/149543496-d950e895-5874-4f65-8ca9-e42572411744.JPG)
[Banks-Meyers and Ray-Tuthill Ancestors.zip](https://github.com/ShammyLevva/FTAnalyzer/files/7871539/Banks-Meyers.and.Ray-Tuthill.Ancestors.zip)
.

**Comment by sbankscharles** [comment]
I was so glad when I discovered your group and started using FTAnalyzer. This is one of a a few groups were everyone’s questions are answered and issues are resolved. 
Thank you so much for looking into this. I really appreciate you explaining the issue and it means so much that you made a fix. I’ll download the update when I get home and check it out. 

Thanks 
Charles

**Comment by ShammyLevva** [comment]
v8.5.2.0 isn't published yet I'm taking a bit of time over weekend to work on some other issues raised recently.

**Comment by sbankscharles** [comment]
Thanks for letting me know.  I did check but also noticed you stated “this is fixed for….” so figured you had not released it.

 

I really appreciate people like you who take time to help strangers out without asking for anything in return.  I’ve noticed a lot of people who work with genealogy are willing to help other people more than any other group on the internet.  Do you know (of) Kevin Labore?  He is starting to work with me to help clean up my tree and eventually put my tree on my own website.

 

Once again, thanks for looking into what I thought was a possible issue with my tree.  I’ve still got enough other weird issues I’m sorting through thanks to FTAnalyzer.

 

Charles

**Comment by ShammyLevva** [comment]
Happy to help. I'm not aware of Kevin.

**Comment by sbankscharles** [comment]
He has posted a lot of responses on your group, which is where I first started corresponding with him.  I did not know at first if he was a moderator or not.  

 

You and him should talk.  He created an amazing Excel Spreadsheet with Power Queries with a connection to Roots Magic 8 database file.  He also has some great insight about using TNG to create a personal ancestry website.  Between the two of you I have been re-energized working with genealogy.  FTM 2019 and one other FB group almost killed my enthusiasm for this.

 

Of course, before I met you both I thought I was a pretty advanced programmer!  LOL
